src/dec_png.c

```diff
@@ -1,3 +1,8 @@
+/**
+ * @file
+ * @brief This struct read and decode PNG stream
```

Comment:
This needs to be a properly punctuated sentence. 

Also there need to be an author tag for the file.